scanpipe/management/commands/graph.py

```diff
@@ -26,19 +26,46 @@
 from django.core.management import CommandError
 from django.core.management.base import BaseCommand
 
+from scanpipe.management.commands import indent
 from scanpipe.management.commands import scanpipe_app_config
-from scanpipe.pipelines import PipelineGraph
 from scanpipe.pipelines import get_pipeline_class
 from scanpipe.pipelines import get_pipeline_doc
 
 
 def is_graphviz_installed():
-    exitcode, _ = subprocess.getstatusoutput("which dot")
+    exitcode = subprocess.getstatusoutput("which dot")[0]
     if exitcode == 0:
         return True
     return False
 
 
+def pipeline_graph_dot(pipeline_class, fontname="Helvetica", shape="record"):
```

Comment:
I wonder if in the future we could eschew using `dot` at all.

Reply:
We can get rid of the feature now we have a way to visualize Pipeline in the UI.